doc/theming.rst

```diff
@@ -310,6 +290,44 @@ Python :mod:`ConfigParser` module) and has the following structure:
   and are accessible from all templates as ``theme_<name>``.
 
 
+.. _distribute-your-theme:
+
+Distribute your theme as a python package
+-----------------------------------------
```

Comment:
I think it would be nice if we provide a template for themes (or quickstart for themes!).